applications/vessel-history/src/features/dataviews/dataviews.utils.ts

```diff
@@ -30,8 +30,16 @@ export const getVesselDataviewInstanceId = (vesselId: string) => `${VESSEL_LAYER
 
 export const getVesselDataviewInstance = (
   vessel: { id: string },
-  { trackDatasetId, infoDatasetId, eventsDatasetsId }: VesselInstanceDatasets
+  { trackDatasetId, infoDatasetId, eventsDatasetsId }: VesselInstanceDatasets,
+  akaVessels: { id: string }[] = []
```

Comment:
what does means aka?

Reply:
It is the acronym for Also Known As, I decided to use it because it's shorter and it'll be part of the query string.